tests/unit/hooks/useIsBlockedToAddFeed.test.ts

```diff
@@ -0,0 +1,114 @@
+import {renderHook} from '@testing-library/react-native';
+import Onyx from 'react-native-onyx';
+import useCardFeeds from '@hooks/useCardFeeds';
+import useIsBlockedToAddFeed from '@hooks/useIsBlockedToAddFeed';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import createRandomPolicy from '../../utils/collections/policies';
+
+const mockPolicyID = '123456';
+
+const delay = (ms: number) =>
+    new Promise((resolve) => {
+        setTimeout(resolve, ms);
+    });
+
+const mockPolicy = {...createRandomPolicy(Number(mockPolicyID), CONST.POLICY.TYPE.TEAM, 'TestPolicy'), policyID: mockPolicyID};
+
+jest.mock('@hooks/useCardFeeds', () => ({
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    __esModule: true,
+    default: jest.fn(),
+}));
+describe('useIsBlockedToAddFeed', () => {
+    beforeEach(async () => {
+        await Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${mockPolicy?.policyID}`, mockPolicy);
+    });
+    it('should return true if collect policy and feed already exists', () => {
```

Comment:
Can you also add a test for Collect policy with no feed yet? 

And also test with Collect policy that has Expensify card feed